implementations/rust/ockam/ockam_api/src/nodes/service.rs

```diff
@@ -117,6 +117,7 @@ pub struct NodeManager {
         BootstrapedIdentityStore<PreTrustedIdentities, AuthenticatedAttributeStorage<LmdbStorage>>,
 }
 
+#[derive(Clone)]
```

Comment:
Interesting, where is that used?

Reply:
Modified the first commit to remove it :+1: